spec/lib/graphql_rails/router_spec.rb

```diff
@@ -150,10 +150,12 @@ class RouterDummyUsersController < GraphqlRails::Controller
     end
 
     describe '#rescue_from' do
-      it 'allows rescuing from errors', :aggregate_failures do
-        expect(router.tap(&:reload_schema).graphql_schema.rescues).to be_empty
-        router.rescue_from(StandardError) { 'ups!' }
-        expect(router.tap(&:reload_schema).graphql_schema.rescues).to include(StandardError)
+      it 'allows rescuing from errors' do
+        expect { router.rescue_from(StandardError) { 'ups!' } }.to(
+          change { router.tap(&:reload_schema).graphql_schema.error_handler.find_handler_for(StandardError) }
+            .from(nil)
+            .to(hash_including(class: StandardError))
+        )
```

Comment:
I know it's merged so it's just an advise for the future. I would suggest to avoid using brackets for `to` part in specs. I find this more readable:

```ruby
expect { router.rescue_from(StandardError) { 'ups!' } }
  .to change { router.tap(&:reload_schema).graphql_schema.error_handler.find_handler_for(StandardError) }
    .from(nil)
    .to(hash_including(class: StandardError))
```

Reply:
Any reasoning? Since i find suggestion/request worse considering this as ruby syntax by community standards.

Reply:
>  ruby syntax by community standards.
which standards are you referring to?

It's not a must. I just wanted to share what could improve code in the future. And yes, this is opinionated (...and better structured).

Reply:
I don't think there can be a proper discussion if you say your opinion is better before providing any arguments

Reply:
I doubt that "readability" can ever be non-opinionated. There will be always someone who prefers one way over another. I just wanted to show my preferences and suggest how this could be improved.  That's alright if you do not agree with me - I respect that.

I gave you my reasoning personally, but I can share it here for better visibility. Here are my **opinionated** arguments, why I think my suggested way is more readable.

My suggested code structure has 3 distinct parts:
```
1) expect <action> # expectation
2) to change <state> # change
3) from <old state> to <new state> # comparison
```

currently merged code structure looks like this:
```
1) expect <action> to( # expectation of change
2)   change <state> # change
3)   from <old state> to <new state>  # comparison
4) ) # end of first `to` / end of expectation
````

So in the current code, all those parts are not that district and you need more effort to understand where expectation ends and where state comparison starts.

Just to make it clear: I'm not forcing you to do any changes. I just wanted to share with you an alternative implementation that I find more readable (:tm:)